openstack/cinder/values.yaml

```diff
@@ -13,6 +13,8 @@ global:
     enabled: false
   imagePullPolicy: IfNotPresent
 
+  cinder_service_user: cinder
```

Comment:
What's that for? There's already a default in the chart.

Reply:
ok I didn't see the value in values.yaml and was concerned it might be used elsewhere.  I was comparing the nova chart to see what it does.  I can clean that up later as I prefer explicit settings in values.yaml.